test/webdriverio/test/test_setup.ts

```diff
@@ -78,23 +75,27 @@ export async function driverSetup() {
 
 /**
  * End the webdriverIO session.
+ *
  * @return A Promise that resolves after the actions have been completed.
  */
 export async function driverTeardown() {
-  await driver.deleteSession();
+  await driver?.deleteSession();
   driver = null;
   return;
 }
 
 /**
  * Navigate to the correct URL for the test, using the shared driver.
- * @param {string} playgroundUrl The URL to open for the test, which should be
+ *
+ * @param playgroundUrl The URL to open for the test, which should be
  *     a Blockly playground with a workspace.
- * @returns {Promise<webdriverio.Browser>} A Promsie that resolves to a webdriverIO browser that tests can manipulate.
+ * @returns A Promsie that resolves to a webdriverIO browser that tests can manipulate.
```

Comment:
Nit: promise

Reply:
Thanks, 1ba7ae46d3719d1e2a2a9dd77a26b3cbfb81a490

(I think this one originates in the same file in core)